mythril/analysis/modules/dependence_on_predictable_vars.py

```diff
@@ -16,172 +17,180 @@ def __init__(self):
         super().__init__(
             name="Dependence of Predictable Variables",
             swc_id="{} {}".format(TIMESTAMP_DEPENDENCE, PREDICTABLE_VARS_DEPENDENCE),
-            hooks=["COINBASE", "GASLIMIT", "TIMESTAMP", "NUMBER", "BLOCKHASH"],
+            hooks=["CALL", "CALLCODE", "DELEGATECALL", "STATICCALL"],
             description=(
                 "Check for CALLs that send >0 Ether as a result of computation "
                 "based on predictable variables such as block.coinbase, "
                 "block.gaslimit, block.timestamp, block.number"
             ),
+            entrypoint="callback",
         )
+        self._issues = []
 
-    def execute(self, statespace):
+    @property
+    def issues(self) -> list:
+        return self._issues
 
+    def execute(self, state: GlobalState) -> list:
         log.debug("Executing module: DEPENDENCE_ON_PREDICTABLE_VARS")
+        self._issues.extend(_analyze_states(state))
+        return self.issues
 
-        issues = []
 
-        for call in statespace.calls:
+detector = PredictableDependenceModule()
+
+
+def _analyze_states(state: GlobalState) -> list:
+    issues = []
+    call = get_call_from_state(state)
+    if call is None:
+        return []
+    if "callvalue" in str(call.value):
+        log.debug("[DEPENDENCE_ON_PREDICTABLE_VARS] Skipping refund function")
```

Comment:
We don't need to print `[DEPENDENCE_ON_PREDICTABLE_VARS]` with named loggers anymore.

Reply:
Oh! cool, it was previously like that, so i left it that way.